### PR TITLE
Binary Quantization Article Updates

### PR DESCRIPTION
**Pull request by NirantK**
I've updated

## 1. Binary Quantization Article
- Added results with dataset links for the OpenAI v3 models, Gemini and Mistral
- Touch ups with changes in CTA and links to Quantization Guide

## 2. BQ with OpenAI v3 Article
- Fixed a lot of minor snafus
- Added OpenAI v3 Results table in markdown

### CODE REVIEW

File: qdrant-landing/content/articles/binary-quantization-openai.md [hunk 1]
```diff
@@ -201,6 +211,11 @@ We recommend the following best practices for leveraging Binary Quantization to
...
+
+Learn more about how to boost your vector search speed and accuracy while reducing costs: [Quantization](https://qdrant.tech/documentation/guides/quantization/?selector=aHRtbCA%2BIGJvZHkgPiBkaXY6bnRoLW9mLXR5cGUoMSkgPiBzZWN0aW9uID4gZGl2ID4gZGl2ID4gZGl2Om50aC1vZi10eXBlKDIpID4gYXJ0aWNsZSA%2BIGgyOm50aC1vZi10eXBlKDIp).
 
-Learn more about how to boost your vector search speed and accuracy while reducing costs: [Binary Quantization.](https://qdrant.tech/documentation/guides/quantization/?selector=aHRtbCA%2BIGJvZHkgPiBkaXY6bnRoLW9mLXR5cGUoMSkgPiBzZWN0aW9uID4gZGl2ID4gZGl2ID4gZGl2Om50aC1vZi10eXBlKDIpID4gYXJ0aWNsZSA%2BIGgyOm50aC1vZi10eXBlKDIp)
+The article gives examples of data sets and configuration you can use to get going. Our documentation covers [adding the data](/documentation/tutorials/bulk-upload/) to your Qdrant instance as well as [creating your indices](/documentation/tutorials/optimize/).
```
Comment (kacperlukawski) [inline qdrant-landing/content/articles/binary-quantization-openai.md:220]: The tutorial linked as *creating your indices* is actually about optimizing for different scenarios.

File: qdrant-landing/content/articles/binary-quantization-openai.md [hunk 2]
```diff
@@ -177,6 +179,14 @@ for combination in dataset_combinations:
     print(acc)
 ```
 
+Here is a selected slice of these results, with `rescore=True`:
+
+|Method|Dimensionality|Test Dataset|Recall|Oversampling|
+|-|-|-|-|-|
+|**Recommended** OpenAI text-embedding-3-large|3072|[DBpedia 1M](https://huggingface.co/datasets/Qdrant/dbpedia-entities-openai3-text-embedding-3-large-3072-1M) | 0.9966|3x|
+|OpenAI text-embedding-3-small|1536|[DBpedia 100K](https://huggingface.co/datasets/Qdrant/dbpedia-entities-openai3-text-embedding-3-small-1536-100K)| 0.9847|3x|
+|OpenAI text-embedding-3-large|1536|[DBpedia 1M](https://huggingface.co/datasets/Qdrant/dbpedia-entities-openai3-text-embedding-3-large-1536-1M)| 0.9826|3x|
```
Comment (kacperlukawski) [inline qdrant-landing/content/articles/binary-quantization-openai.md:188]: Why do we mark just one of the entries as recommended? `0.9826` might also be acceptable in many cases.

File: qdrant-landing/content/articles/binary-quantization.md [hunk 3]
```diff
@@ -212,8 +210,15 @@ If you're working with OpenAI or Cohere embeddings, we recommend the following o
...
 |Method|Dimensionality|Test Dataset|Recall|Oversampling|
 |-|-|-|-|-|
-|OpenAI text-embedding-ada-002|1536|[DbPedia](https://huggingface.co/datasets/KShivendu/dbpedia-entities-openai-1M) 1M|0.98|4x|
+|**Recommended** OpenAI text-embedding-3-large|3072|[DBpedia 1M](https://huggingface.co/datasets/Qdrant/dbpedia-entities-openai3-text-embedding-3-large-3072-1M) | 0.9966|3x|
+|OpenAI text-embedding-3-small|1536|[DBpedia 100K](https://huggingface.co/datasets/Qdrant/dbpedia-entities-openai3-text-embedding-3-small-1536-100K)| 0.9847|3x|
+|OpenAI text-embedding-3-large|1536|[DBpedia 1M](https://huggingface.co/datasets/Qdrant/dbpedia-entities-openai3-text-embedding-3-large-1536-1M)| 0.9826|3x|
 |Cohere AI embed-english-v2.0|4096|[Wikipedia](https://huggingface.co/datasets/nreimers/wikipedia-22-12-large/tree/main) 1M|0.98|2x|
+|OpenAI text-embedding-ada-002|1536|[DbPedia 1M](https://huggingface.co/datasets/KShivendu/dbpedia-entities-openai-1M) |0.98|4x|
+|Gemini|768|No Open Data| 0.9563|3x|
+|Mistral Embed|768|No Open Data| 0.9445 |3x|
```
Comment (kacperlukawski) [inline qdrant-landing/content/articles/binary-quantization.md:219]: Again, I'd avoid stating that one of the methods is recommended. Many of them sound reasonable.

File: qdrant-landing/content/articles/binary-quantization.md [hunk 4]
```diff
@@ -212,8 +210,15 @@ If you're working with OpenAI or Cohere embeddings, we recommend the following o
...
+
+NA indicates that the dataset was not released.
```
Comment (kacperlukawski) [inline qdrant-landing/content/articles/binary-quantization.md:221]: *NA* does not occur in the table above.

**Review by kacperlukawski — CHANGES_REQUESTED** [review]

File: qdrant-landing/content/articles/binary-quantization-openai.md (hunk 2 above)
Comment (NirantK) [reply qdrant-landing/content/articles/binary-quantization-openai.md:188]: Fair point, I'll update it to mention that this has the highest MTEB score among those mentioned.

File: qdrant-landing/content/articles/binary-quantization.md (hunk 4 above)
Comment (NirantK) [reply qdrant-landing/content/articles/binary-quantization.md:221]: Thanks for catching this!

File: qdrant-landing/content/articles/binary-quantization.md (hunk 3 above)
Comment (NirantK) [reply qdrant-landing/content/articles/binary-quantization.md:219]: Removing the Recommended!

File: qdrant-landing/content/articles/binary-quantization-openai.md (hunk 1 above)
Comment (NirantK) [reply qdrant-landing/content/articles/binary-quantization-openai.md]: Fixed this link

**Review by sabrinaaquino — APPROVED** [review]

**Pull request merged**